neo/Network/P2P/Payloads/Header.cs

```diff
@@ -51,5 +54,14 @@ public TrimmedBlock Trim()
                 Hashes = new UInt256[0]
             };
         }
+
+        public new static Header FromJson(JObject json)
+        {
+            Header header = new Header();
+            BlockBase blockBase = header;
+            blockBase.FromJson(json);
```

Comment:
((BlockBase)header).FromJson

Reply:
![image](https://user-images.githubusercontent.com/4393234/61856476-cdd43300-aef4-11e9-95ee-6b48789beb48.png)

Reply:
@lock9 what is wrong here?